### DISCUSSION TITLE
IDH node installs, but won't configure.

### DISCUSSION THREAD

**Discussion by zrieprakis** [Security-Onion-Solutions/securityonion#12952] [Unsupported Versions] [answered]
**Version**
2.4.60

**Installation Method**
Security Onion ISO image

**Description**
IDH node installs, but won't configure.

**Installation Type**
Distributed

**Location**
on-prem with Internet access

**Hardware Specs**
Exceeds minimum requirements

**CPU**
For IDH node, 2

**RAM**
For IDH node, 4GB

**Storage**
80

**Network Traffic Speeds**
1Gbps

**Status**
Yes, all services on all nodes are running OK

**Salt Status**
No, there are no failures

**Logs**
Unsure which to provide, don't want to send useless logs.

**Details:**
SO node is installed as standalone, IDH node installed afterwards as distributed.  Both installations finish without errors.  Nodes all show up as OK in Grid after normal reboots.  Config is done on firewall and services that IDH is to provide (HTTP and SSH for now).  Dashboard shows all green, but IDH gives empty response for http requests and management SSH did not move to 2222 - I can access the IDH node from any endpoint (not just the SO management node) via 22.  Multiple reinstalls just to make sure I didn't miss anything.  Anything I can look at or provide to see what's up here?

Thanks in advance.

**Comment by Meftian** [comment]
Same problem here, I have the exact same setup and behavior.

**Comment by reyesj2** [comment]
From your manager /standalone can you run the following and provide the output?

```salt -C "G@role:so-idh" state.highstate -linfo```

**Reply by Meftian** [reply]
Hi, as I have the same behavior, here is my output. In lines 1217 to 1221 you can see errors, that the "idh_sshd_selinux" policy is not found in the SLS 'idh.openssh.config'.
`ID: idh_sshd_selinux`
`Function: selinux.port_policy_present`
`Result: False`
`Comment: State 'selinux.port_policy_present' was not found in SLS 'idh.openssh.config'`
    `Reason: 'selinux' __virtual__ returned False: selinux module could not be loaded`
     `Changes:`

[salt-output.txt](https://github.com/Security-Onion-Solutions/securityonion/files/15271670/salt-output.txt)

**Comment by zrieprakis** [comment]
> From your manager /standalone can you run the following and provide the output?
> 
> `salt -C "G@role:so-idh" state.highstate -linfo`

Interesting.  I have 1 error and 1 warning, the former is the same as @Meftian.  L:1217-1229
[salt-output.txt](https://github.com/Security-Onion-Solutions/securityonion/files/15276909/salt-output.txt)

----------
          ID: idh_sshd_selinux
    Function: selinux.port_policy_present
      Result: False
     Comment: State 'selinux.port_policy_present' was not found in SLS 'idh.openssh.config'
              Reason: 'selinux' __virtual__ returned False: selinux module could not be loaded
     Changes:
----------
          ID: openssh_config
    Function: file.replace
        Name: /etc/ssh/sshd_config
      Result: None
     Comment: Changes would have been made
     Changes:
----------

**Comment by m0duspwnens** [comment, marked as answer]
An [issue](https://github.com/Security-Onion-Solutions/securityonion/issues/12991) has been created to track this and will be included in the 2.4.70 release. In the meantime, if you have this issue, you can run the follow two commands on the IDH node. `sudo yum install policycoreutils-python-utils` , then `sudo salt-call state.highstate`
